### PR TITLE
use input event listener for word counter example closes #51

### LINKED ISSUE #51: Falsy comment in the word-count-web-component javascript file.
In the word-count-web-component javascript file, there is a comment that goes as follows :

```js
// Update count when element content changes
    setInterval(function() {
      var count = 'Words: ' + countWords(wcParent);
      text.textContent = count;
    }, 200) 
```

This comments proves to be falsy, because the code running under it does not apply the new word count "when" the content changes. It applies the new word count every 200 milliseconds. 

To make the code reflect the comment in a more truthful way, the code should instead listen for the input event on the parent node (the article itself) since the whole article is made editable via the contenteditable attribute in the html file.

it should go like this instead : 

```js
// Update count when element content changes
this.parentNode.addEventListener('input', () => {
      var count = 'Words: ' + countWords(wcParent);
      text.textContent = count;
    });
```

This way, the code respects the comment more and performance is also better (even if this is a small demo, those principles should be important to always consider. Even more, since this is MDN and MDN is a reference for alot of dev coming from alot of ranges of experience)

### PR DESCRIPTION
**Pull request by MiB-1**
Word counter example
As mentioned in the issue instead of using setInterval we can use input event listener.

Closes #51

### CODE REVIEW

**Review by ralyodio — APPROVED** [review]

**Comment by bsmth** [comment]
Thanks a lot for opening this one. BTW, we have the following in content that also needs to be updated: https://github.com/mdn/content/blob/ca1fc680cd7c7c04bed18d8ab1fa3988bf7e972e/files/en-us/web/api/element/attachshadow/index.md?plain=1#L163-L166

**Review by bsmth — APPROVED** [review]
Nice, works well locally after checking out the branch 👍🏻

**Comment by MiB-1** [comment]
thanks @bsmth , here is the link for updating the content repo.
[https://github.com/mdn/content/pull/36540](url)

File: word-count-web-component/main.js [hunk 1]
```diff
@@ -25,10 +25,9 @@ class WordCount extends HTMLParagraphElement {
     shadow.appendChild(text);
 
     // Update count when element content changes
-    setInterval(function() {
-      const count = `Words: ${countWords(wcParent)}`;
-      text.textContent = count;
-    }, 200);
+    this.parentNode.addEventListener('input', () => {
+      text.textContent = 'Words: ' + countWords(wcParent);
```
Comment (Josh-Cena) [inline word-count-web-component/main.js:29]:
```suggestion
      text.textContent = `Words: ${countWords(wcParent)}`;
```

Comment (MiB-1) [reply word-count-web-component/main.js]: thanks for the suggestion @Josh-Cena . updated the code.

**Review by bsmth — APPROVED** [review]
Super, thanks a lot!

**Comment by MiB-1** [comment]
> Super, thanks a lot!

thanks a lot, looking forward to contribute more. may I know when this be PR will get merge?

**Comment by bsmth** [comment]
Merging now!

**Pull request merged**